src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -748,20 +752,90 @@ const UsersConfigurationRow = ({
     );
 };
 
+// This method needs to be outside of component as re-render would create a new instance of debounce
+const validateKey = debounce(500, (key, setKeyObject) => {
+    if (isEmpty(key))
+        return;
+
+    return cockpit.script(`echo ${key} | ssh-keygen -l -f /dev/stdin`, { error: "message" })
```

Comment:
This is dangerous, if `key` contains anything funny, like newlines or spaces. Please use `cockpit.spawn()` and `.input(key)`. Please also use the more common `-` instead of `/dev/stdin`.